tests/regression/runner.py

```diff
@@ -0,0 +1,121 @@
+# It would not be easy to evaluate precise preformance
+# Python3
+import json
+import requests
+import sys
+import os
+
+if len(sys.argv) == 2:
+    raw_url = sys.argv[1]
+else:
+    raw_url = "localhost:5566"
+
+url = "http://" + raw_url
+headers = {'content-type': 'application/json'}
+
+# Utils:
+TIMEOUT = 100  # [sec]
+
+
+def API(get_query, get_data=None, post_data=None):
+    try:
+        if get_data is not None:
+            r = requests.get(str(url + get_query + get_data),
+                             timeout=TIMEOUT)
+            if r.status_code == 405:
+                response = "[405] Method Not Allowed"
+            else:
+                response = r.text
+        elif post_data is not None:
+            stringified = json.dumps(post_data)
+            r = requests.post(str(url + get_query),
+                              data=stringified,
+                              headers=headers,
+                              timeout=TIMEOUT)
+            response = r.text
+        else:
+            print("Wrong request method")
+            return -1
+
+    except BaseException:
+        print(url, "Timeout!")
+        print('\n    ' + repr(sys.exc_info()))
+        return "\n [failed] \n"
+    if not response:
+        response = ""
+    return response
+
+
+def mam_send_msg():
+    """ cmd
+        1. Eng char only msg [success]
+        1. ascii symbols msg [success]
+        2. chinese msg [success] unicode string can send msg successfully,
+                                 since python transfer it into utf8 already
+        3. japanese msg [success]
+    """
+    cmds = [
+        "ToBeOrNotToBe", "I met my soulmate. She didn’t", "當工程師好開心阿",
+        "今夜は月が綺麗ですね"
+    ]
+
+    [x.encode('utf-8') for x in cmds]
+    response = []
+
+    for cmd in cmds:
+        print("cmd = " + cmd)
+        cmd_dict = {"message": cmd}
+        response.append(API("/mam/", post_data=cmd_dict))
+
+    return response
+
+
+def mam_recv_msg():
+    """ cmd
+        1. Correct exist MAMv2 msg [success]
+        2. Empty [failed] empty parameter causes http error 405
+    """
+    cmds = [
+        "QBFXQETKSHDYPFUDO9ILVCAVQIXOHXKCECZYFLPBNVIX9JUXQZJE9URQEEUWPWYZOIACTCGZX9IDIODCA",
+        ""
+    ]
+    response = []
+    for cmd in cmds:
+        print("cmd = " + cmd)
+        response.append(API("/mam/", get_data=cmd))
+
+    return response
+
+
+def main():
+    """
+    API List
+        mam_recv_msg: GET
+        mam_send_msg: POST
+        Find transactions by tag
+        Get transaction object
+        Find transaction objects by tag
+        Get transaction object
+        Find transaction objects by tag
+        Fetch pair tips which base on GetTransactionToApprove
+        Fetch all tips
+        Generate an unused address
+        send transfer
+        Client bad request
+    """
+    filepath = os.path.dirname(sys.argv[0]) + "/mam-result.log"
+    f = open(filepath, "w")
+    res_s = mam_send_msg()
```

Comment:
Add description to different apis call with comment since these statements would grow longer.
Like 
```
# mam send message
res_s = mam_send_msg()
...
```